cmd/sops/common/common.go

```diff
@@ -139,19 +170,40 @@ func IsIniFile(path string) bool {
 	return strings.HasSuffix(path, ".ini")
 }
 
+// DefaultStoreForFormat returns the correct format-specific implementation
+// of the Store interface given the format string
+func DefaultStoreForFormat(format string) Store {
+	storeConst, found := StoreConstructors[format]
+	if !found {
+		storeConst = StoreConstructors["binary"] // default
+	}
+	return storeConst()
+}
+
 // DefaultStoreForPath returns the correct format-specific implementation
 // of the Store interface given the path to a file
 func DefaultStoreForPath(path string) Store {
```

Comment:
We should instead return a `Format` for a given path.

Reply:
1. this would not be backwards compatible.
2. how do you want me to name it? suggestions?

Reply:
This is not part of the public API, so it's okay to not be backwards compatible. `FormatForPath` would be good, I think.